pkg/authorization/registry/resourceaccessreview/rest.go

```diff
@@ -63,7 +63,9 @@ func (r *REST) Create(ctx apirequest.Context, obj runtime.Object, _ bool) (runti
 	}
 
 	attributes := util.ToDefaultAuthorizationAttributes(nil, resourceAccessReview.Action.Namespace, resourceAccessReview.Action)
-	users, groups, err := r.subjectLocator.GetAllowedSubjects(attributes)
+	subjects, err := r.subjectLocator.AllowedSubjects(attributes)
```

Comment:
Add comments for places where its OK to "ignore" errors.

Reply:
You cannot change the interface, but you should add a comment when you use a function and ignore the error, like here, on why it is ok to ignore the error.

Reply:
@simo5 but we actually do not ignore this error, we check it here: https://github.com/abstractj/origin/blob/e4e509dc61fcf944805869c8dbd9ab63f2438d54/pkg/authorization/registry/resourceaccessreview/rest.go#L80-L82. 

I don't mind to add comments, but I feel like I'm missing the point.

Reply:
Something like:

```go
func (r *REST) Create(ctx apirequest.Context, obj runtime.Object, _ bool) (runtime.Object, error) {
	resourceAccessReview, ok := obj.(*authorizationapi.ResourceAccessReview)
	if !ok {
		return nil, kapierrors.NewBadRequest(fmt.Sprintf("not a resourceAccessReview: %#v", obj))
	}
	if errs := authorizationvalidation.ValidateResourceAccessReview(resourceAccessReview); len(errs) > 0 {
		return nil, kapierrors.NewInvalid(authorizationapi.Kind(resourceAccessReview.Kind), "", errs)
	}

	user, ok := apirequest.UserFrom(ctx)
	if !ok {
		return nil, kapierrors.NewInternalError(errors.New("missing user on request"))
	}

	// if a namespace is present on the request, then the namespace on the on the RAR is overwritten.
	// This is to support backwards compatibility.  To have gotten here in this state, it means that
	// the authorizer decided that a user could run an RAR against this namespace
	if namespace := apirequest.NamespaceValue(ctx); len(namespace) > 0 {
		resourceAccessReview.Action.Namespace = namespace

	} else if err := r.isAllowed(user, resourceAccessReview); err != nil {
		// this check is mutually exclusive to the condition above.  localSAR and localRAR both clear the namespace before delegating their calls
		// We only need to check if the RAR is allowed **again** if the authorizer didn't already approve the request for a legacy call.
		return nil, err
	}

	attributes := util.ToDefaultAuthorizationAttributes(nil, resourceAccessReview.Action.Namespace, resourceAccessReview.Action)

	// err is non fatal and partial success is possible
	subjects, err := r.subjectLocator.AllowedSubjects(attributes)
	// expandSubjectError is non fatal and partial success is possible
	users, groups, expandSubjectError := authorizationutil.ExpandSubjects(resourceAccessReview.Action.Namespace, subjects)

	response := &authorizationapi.ResourceAccessReviewResponse{
		Namespace: resourceAccessReview.Action.Namespace,
		Users:     users,
		Groups:    groups,
	}

	if err != nil {
		response.EvaluationError = err.Error()
	}
	if err != nil {
		response.EvaluationError += " " + expandSubjectError.Error()
	}

	return response, nil
}
```